src/test/kotlin/no/nav/pdfgen/HtmlTemplates.kt

```diff
@@ -4,3 +4,6 @@ import org.apache.pdfbox.io.IOUtils
 
 val testTemplateIncludedFonts: String = IOUtils.toByteArray(PdfGenITSpek::class.java.getResourceAsStream("/html/test_template_included_fonts.html")).toString(Charsets.UTF_8)
 val testTemplateInvalidFonts: String = IOUtils.toByteArray(PdfGenITSpek::class.java.getResourceAsStream("/html/test_template_invalid_fonts.html")).toString(Charsets.UTF_8)
+
+val testJpg: ByteArray = IOUtils.toByteArray(PdfGenITSpek::class.java.getResourceAsStream("/image/test.jpg"))
```

Comment:
Could avoid using external dependency by using stdlib function (perhaps even extract to a function):
```kotlin
val testJpg: ByteArray = PdfGenITSpek::class.java.getResourceAsStream("/image/test.jpg").readBytes()
val testPng: ByteArray = PdfGenITSpek::class.java.getResourceAsStream("/image/test.png").readBytes()
```